src/cmd/viper.go

```diff
@@ -117,7 +117,7 @@ func initViper() {
 	if err != nil {
 		// Config file not found; ignore
 		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			message.Error(err, lang.CmdViperErrLoadingConfigFile)
+			message.Error(err, lang.CmdViperErrLoadingConfigFile, err)
```

Comment:
```suggestion
			message.Error(err, fmt.Sprintf(lang.CmdViperErrLoadingConfigFile, err.Error()))
```

NOTE: This will require importing `fmt` in this package and modifying the `lang.CmdViperErrLoadingConfigFile` variable to be `%s` instead of `%w`